tool/site/client-react/src/pages/sendPostSignPage.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react"
+import {useContext} from "react"
+import {Button, CrossIcon, Label, SummaryList, Table, TickIcon} from "nhsuk-react-components"
+import axios from "axios"
+import * as fhir from "fhir/r4"
+import MessageExpanders from "../components/messageExpanders"
+import ButtonList from "../components/buttonList"
+import LongRunningTask from "../components/longRunningTask"
+import {AppContext} from "../index"
+import PrescriptionActions from "../components/prescriptionActions"
+
+interface SendPostSignPageProps {
+  token: string
+}
+
+const SendPostSignPage: React.FC<SendPostSignPageProps> = ({
+  token
+}) => {
+  const {baseUrl} = useContext(AppContext)
+  const sendPrescriptionTask = () => sendPrescription(baseUrl, token)
+  return (
+    <LongRunningTask<SendResult | SendBulkResult> task={sendPrescriptionTask} loadingMessage="Sending prescription(s).">
```

Comment:
can determine if there's one or many prescriptions using isBulk

Reply:
We don't have the result at the point we're rendering the message. We _could_ get it from the request, but I don't think it matters enough to bother